persistence/ravendb/clusterconfiguration.md

```diff
@@ -0,0 +1,35 @@
+
+To provide some context on what that means practically speaking, to enable the usage of cluster wide transactions it's required to:
+- Modify all existing documents to store a synchronization identifier. This can be part of the document, or can be stored inside the document’s metadata.
+- Create a corresponding compare exchange value for every document. The value of the compare exchange value needs to be set to the synchronization identifier’s value.
+Once that’s done, in order to update a document it's needed to:
+- Load the document (and its meta data if needed)
+- Load the corresponding compare exchange value
+- Compare the document’s synchronization identifier with the one stored in the compare exchange value that corresponds to the document. If they are not the same, it means that the document has already been changed and both documents need to be reloaded.
+- Update the document as well as the synchronization identifier
+- Update the compare exchange value’s document value with the new synchronization identifier
+
+Given the complexity involved in implementing cluster wide transactions properly across the business data and the persistence data that’s stored by NServiceBus, it was decided to stop supporting clusters in RavenDB for now. Why? The RavenDB persistence can use cluster wide transactions to correctly and safely store NServiceBus data, but the persistence can’t make sure any business data modified in an NServiceBus handler is safe. Each handler would need to be reassessed using the steps listed above to ensure correct usage of cluster-wide transactions.
+
+Not using cluster-wide transactions in the correct form for the business data would result in possible data loss that is impossible to detect. Given that it may surface as possible message loss, it was decided to withdraw support for clusters. Unless cluster wide transactions were already used in the correct form, introducing them comes with a very severe impact on the system.
+
+## How to move forward
+
+- If the RavenDB server is configured in a cluster, move back to using a single node. This is less than ideal, as it introduces a single point of failure in the system.
+- Move to another persistence option. This would require migrating data from RavenDB to [another persistence option](/persistence).
+
+Don't hesitate to [contact support](https://particular.net/support) if faced with this scenario.
+
+## What about ServiceControl
+
+ServiceControl uses an embedded RavenDB database. However, there is no impact for ServiceControl users as it still relies on RavenDB 3.5 which makes use of ESENT.
```

Comment:
ESENT doesn't have anything to do with anything. Nor do we need to call out the version.
```suggestion
ServiceControl uses an embedded RavenDB database that is not affected.
```